dispatch/route.go

```diff
@@ -158,6 +162,9 @@ type RouteOpts struct {
 	// What labels to group alerts by for notifications.
 	GroupBy map[model.LabelName]struct{}
 
+	// Use all alert's labels to group
```

Comment:
```suggestion
	// Use all alert labels to group.
```

Not sure we need the `'s` here.